project-base/tests/App/Functional/Model/Transport/IndependentTransportVisibilityCalculationTest.php

```diff
@@ -26,7 +26,7 @@ class IndependentTransportVisibilityCalculationTest extends TransactionFunctiona
     private $independentTransportVisibilityCalculation;
 
     /**
-     * @var \Shopsys\FrameworkBundle\Model\Transport\TransportDataFactoryInterface
```

Comment:
Why is it necessary to replace interfaces with class?

Reply:
Because it is creating `TransportData` which are passed into project-base's `Transport`.

I chosed to change annotation of that factory then annotating all related variables. As it is test in project-base where the `TransportDataFactory` is overiden it is better way to me.